tracer/src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -52,6 +52,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.DiaSymReader.Native" Version="1.7.0" />
```

Comment:
this is not required in this PR right?

Reply:
Right! Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/2592/commits/5814bd9fa43effb3c1d6176a2d64c0450158b7ba.